docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -1,63 +1,63 @@
 === macOS Server
 
-This is only required for building iOS applications.
+NOTE: This is required only if you want to build iOS applications.
 
-==== macOS Machine Setup
+==== macOS and macOS Ansible Scripts Configuration
 
-- Set `ansible_sudo_pass` variable for the Ansible job to enable the Installer to work with root permissions. It is required by several installation steps.
+- Set the password in `ansible_sudo_pass` variable to give the Installer `sudo` privileges.
 
 .macOS Scripts: https://github.com/aerogear/digger-jenkins/blob/master/admin/create-osx-user.sh[create-osx-user.sh] https://github.com/aerogear/digger-jenkins/blob/master/admin/enable-osx-remote-login.sh[enable-osx-remote-login.sh]
 - Set up a user with sudo permissions and SSH access.
 These scripts will create a user named `jenkins` with SSH access.
-It is highly recommended that you read these scripts and change the `PASSWORD`
-variable in `create-osx-user.sh` to a strong passphrase:
-    
+It is highly recommended to set a strong `PASSWORD` in `create-osx-user.sh`.
 
 --
   
-. Download the scripts into the target macOS machine:
+. Download the scripts to the macOS system:
 +
 [source,bash]
 ----
 curl -O https://raw.githubusercontent.com/aerogear/digger-jenkins/master/admin/create-osx-user.sh
 curl -O https://raw.githubusercontent.com/aerogear/digger-jenkins/master/admin/enable-osx-remote-login.sh
 ----
 +
-. Edit the `PASSWORD` for the Jenkins user in `create-osx-user.sh`.
-. Execute `create-osx-user.sh` with the updated password:
+. Set a secure user `PASSWORD` in `create-osx-user.sh`.
+. Run `create-osx-user.sh`:
 +
 ----
 sudo bash create-osx-user.sh
 ----
 +
-. Execute `enable-osx-remote-login.sh` to allow SSH access:
+. Run `enable-osx-remote-login.sh` to enable SSH access:
 +
```

Comment:
I think most of the above is covered in the new pr

Reply:
:+1